### PR TITLE
Add Smiley to dummy app, an example of a compound component

### PR DESCRIPTION
**Pull request by kagemusha**

### CODE REVIEW

File: tests/dummy/app/components/smiley/component.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+export default class SmileyComponent extends Component {
+  @tracked x = this.args.x;
+  @tracked y = this.args.y;
+  @tracked radius = this.args.radius;
```
Comment (mjanjic01) [inline tests/dummy/app/components/smiley/component.js]: These properties don't need to be tracked, as they are only assigned once and never change.
In the getters you can use `this.args.x` directly

File: tests/dummy/app/components/smiley/template.hbs [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<@parent.Circle @x={{@x}} @y={{@y}} @radius={{@radius}} @fill={{@back}} @stroke={{@fore}} @strokeWidth={{2}} @draggable={{true}} />
+<@parent.Circle @x={{leftEyeX}} @y={{eyeY}} @radius={{4}} @fill={{@fore}} />
+<@parent.Circle @x={{rightEyeX}} @y={{eyeY}} @radius={{4}} @fill={{@fore}} />
+<@parent.Arc @x={{@x}} @y={{mouthY}} @rotation={{70}} @innerRadius={{40}} @outerRadius={{43}} @angle={{40}} @fill={{@fore}} />
+<@parent.Text @text={{@name}} @x={{nameX}} @width={{nameWidth}} @y={{nameY}} @align="center" />
```
Comment (mjanjic01) [inline tests/dummy/app/components/smiley/template.hbs]: To make the entire thing draggable you would need to have a single Konva node to group them together, therefor a `Group`.

Suggestion:
```suggestion
<@parent.Group @draggable={{true}} as |G|>
  <G.Circle @x={{@x}} @y={{@y}} @radius={{@radius}} @fill={{@back}} @stroke={{@fore}} @strokeWidth={{2}} />
  <G.Circle @x={{leftEyeX}} @y={{eyeY}} @radius={{4}} @fill={{@fore}} />
  <G.Circle @x={{rightEyeX}} @y={{eyeY}} @radius={{4}} @fill={{@fore}} />
  <G.Arc @x={{@x}} @y={{mouthY}} @rotation={{70}} @innerRadius={{40}} @outerRadius={{43}} @angle={{40}} @fill={{@fore}} />
  <G.Text @text={{@name}} @x={{nameX}} @width={{nameWidth}} @y={{nameY}} @align="center" />
</@parent.Group>
```

Note the `@draggable` is moved from the first circle and onto the group

File: package.json [hunk 3]
```diff
@@ -57,7 +58,6 @@
     "qunit": "^2.16.0",
     "qunit-dom": "^1.6.0"
   },
-  "peerDependencies": {},
```
Comment (mjanjic01) [inline package.json:60]: Also add konva versions `>=7.1.4` (the earliest version I tested with) into the peer dependencies.
So you can provide your own konva version in the project consuming this addon.
```
"peerDependencies": {
  "konva": ">=7.1.4"
}
```

**Review by mjanjic01 — CHANGES_REQUESTED** [review]

**Comment by kagemusha** [comment]
made edits.  let me know if all good

**Comment by mjanjic01** [comment]
Looks good 👍

**Comment by kagemusha** [comment]
awesome and thanks for authoring this addon

**Pull request merged**